website/blog/2022-03-30-version-068.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Announcing React Native 0.68
+authors: [kelset, shikaSD]
+tags: [announcement, release]
+---
+
+# Announcing 0.68
+
+Hello everyone! Today we are announcing the 0.68.0 release of React Native, with opt-in to the New React Native Architecture, bug fixes and more.
+
+### Sections
+
+- [Highlights of 0.68](/blog/2022/03/30/version-068#highlights-of-068)
+- [Opting in into the New Architecture](/blog/2022/03/30/version-068#opting-in-into-the-new-architecture)
+- [Website updates](/blog/2022/03/30/version-068#website-updates)
+- [Interested in helping React Native stabilise new releases?](/blog/2022/03/30/version-068#interested-in-helping-react-native-stabilize-new-releases)
+
+<!--truncate-->
+
+## Highlights of 0.68
+
+[Andrei Calazans](https://twitter.com/Andrei_Calazans) helped us selecting the most relevant changes that 0.68 brings along:
+
+### Breaking changes and version bumps
+
+This version brings along a few breaking changes:
+
+- React Native has been updated to Node 16, the latest LTS. Since on CI we test for LTS and the previous LTS, this change means that users are now required to use a version of Node >= 14.
+- Android Gradle Plugin was updated to 7.0.1, enforcing JDK 11 for Android builds, so make sure to upgrade your configurations (we recommend you use the `temurin11` JDK flavor)
+- Removed `fallbackResource` from `RCTBundleURLProvider` API on iOS.
```

Comment:
Maybe expand on this that its safe to just remove that parameter from the method call without any replacement.